docs/FAQ.md

```diff
@@ -1,9 +1,27 @@
 # FAQ for s2n-tls
 
+### Why is my connection using TLS1.0?
+It is possible that your security policy allows TLS1.0 and your peer only
```

Comment:
I think we should start off with the assumption that the user is reading the packet capture wrong, as I feel like that's the kind of question we usually respond to. How many clients actually only support 1.0?

Reply:
haha starting off with "you're probably wrong" felt a little rude :) I'd like to keep the quick concession that the reader might not be asking the wrong question.

Reply:
Alright I guess I'm just the mean one.